mbed_app.json

```diff
@@ -324,6 +325,11 @@
             "help"      : "Helper macro to enable calculation of rom regions. target.header_offset and target.app_offset still needs to be calculated manually, though.",
             "value"     : "(32*1024)",
             "macro_name": "MBED_BOOTLOADER_SIZE"
+        },
+        "run-ci": {
+            "help"           : "Configuration item to control what verification is done to the target. 0=nothing, 1=build, 2=build+test",
```

Comment:
Proposal:

The level of CI verification level.

0 = ignored
1 = build-only
2 = build + E2E tests